ONSdigital/eq-author-prototypes
# Added hot reload

Re-enabled hot reloading of components to prevent a full page refresh when a component is saved.

## How to test

- load app
- change a component and save
- page should not refresh but component should update
